completion/zsh/_task

```diff
@@ -55,13 +63,13 @@ _task() {
         '(-t --taskfile)'{-t,--taskfile}'[specify a different taskfile]:taskfile:_files' \
         '(-v --verbose)'{-v,--verbose}'[verbose mode]' \
         '(-w --watch)'{-w,--watch}'[watch-mode for given tasks, re-run when inputs change]' \
+        '(operation)*: :__task_list' \
         + '(operation)' \
-            {-l,--list}'[list describable tasks]' \
-            {-a,--list-all}'[list all tasks]' \
-            {-i,--init}'[create new Taskfile.yml]' \
-            '(-*)'{-h,--help}'[show help]' \
-            '(-*)--version[show version and exit]' \
-            '*: :__task_list'
+            '(*)'{-l,--list}'[list describable tasks]' \
+            '(*)'{-a,--list-all}'[list all tasks]' \
+            '(*)'{-i,--init}'[create new Taskfile.yml]' \
+            '(- *)'{-h,--help}'[show help]' \
+            '(- *)--version[show version and exit]' \
```

Comment:
```diff
- (-*)--version
+ (- *)--version
```

I suppose `(- *)` is the intended one. This prevents all options and positional arguments from being suggested when the `--version` option is supplied.

(The parentheses specify groups of arguments to prevent suggestions. [`-` and `*` are special group names for options and positional arguments, respectively](https://zsh.sourceforge.io/Doc/Release/Completion-System.html#index-_005farguments). I think the `-*` group is not defined here.)